src/app/util/util.js

```diff
@@ -42,3 +42,28 @@ NavPill.propTypes = {
   to: PropTypes.string,
   children: PropTypes.node
 };
+
+/**
+ * Dates
+ */
+import {format} from 'date-fns';
+import * as locales from 'date-fns/locale'; // we are ok with importing all languages
+/**
+ * Call's `date-fns/format` with a locale from `navigator.language`
+ *
+ * @param {Date | number | string} date - a date to format
+ * @param {string?} formatStyle - a format string (eg 'DD.MM.YYY'), default is 'P'
```

Comment:
string? ?

Reply:
It comes from Typescript and means `string | null`. My IDE understands that. Let me perhaps change it to ` | undefined`